internal/pkg/deploy/cloudformation/stack/env_integration_test.go

```diff
@@ -32,8 +32,11 @@ name: test
 type: Environment
 # Create the public ALB with certificates attached.
 # All these comments should be deleted.
+cdn: true
 http:
   public:
+    ingress:
```

Comment:
Would you mind syncing with @Lou1415926 to make sure the security groups experience is consistent with her changes: https://github.com/aws/copilot-cli/pull/3763/files#diff-0fc2e38bf10b33570f95dbb18d51eb694ff20caa1e87860c18c32d800a6b7266?

Just reading that scheme it sounds like a more consistent format for the two changes is (feel free to change it):
```yaml
http:
  public:
     security_groups:
        restrict_from_cdn: true
```